boot/zephyr/Kconfig

```diff
@@ -72,8 +72,10 @@ choice BOOT_SIGNATURE_TYPE
 	default BOOT_SIGNATURE_TYPE_RSA
 
 config BOOT_SIGNATURE_TYPE_NONE
-	bool "No signature; use only hash check"
-	select BOOT_USE_TINYCRYPT
+	bool "No signature; use only sha256 check"
+
+config BOOT_SIGNATURE_TYPE_NONE_SHA512
+	bool "No signature; use only sha512 check"
```

Comment:
Not really sure what the point of this change is? The signature would only be for detecting bit flips, the chance of having a bit flip and a sha256 hash still matching is basically impossible, so not sure why allowing a sha512 for this would improve anything?

Reply:
In general yes, but it's security requirement we got.

Reply:
AFAIK  `BOOT_SIGNATURE_TYPE_NONE` was introduced for giving SW evaluation (for development testing) value. It is not intended to use in any real product. If we are not going to use `BOOT_SIGNATURE_TYPE_NONE_SHA512` for anything like this - then it is not needed.